tests/test_studies.py

```diff
@@ -0,0 +1,6 @@
+import pytest
+
+@pytest.mark.usefixtures("authenticated_request")
```

Comment:
Add a test without the authentication fixture because that's the most important: restrict access to no logged user